packages/components/src/presentation-components/properties/inputs/UseQuantityValueInput.tsx

```diff
@@ -102,40 +110,37 @@ export function useQuantityValueInput({ initialRawValue, schemaContext, koqName
     inputProps: {
       onChange,
       placeholder,
-      value: quantityValue.formattedValue,
-      disabled: !formatter || !parser,
+      disabled: !highPrecisionFormatter || !parser,
     },
   };
 }
 
 function useFormatterAndParser(koqName: string, schemaContext: SchemaContext) {
   interface State {
-    placeholderFormatter?: FormatterSpec;
-    formatterSpec: FormatterSpec;
+    defaultFormatter?: FormatterSpec;
+    highPrecisionFormatter: FormatterSpec;
     parserSpec: ParserSpec;
   }
-
   const [state, setState] = useState<State>();
 
   useEffect(() => {
     const findFormatterAndParser = async () => {
-      // eslint-disable-next-line @typescript-eslint/no-deprecated
-      const koqFormatter = new KoqPropertyValueFormatter(schemaContext, undefined, IModelApp.formatsProvider);
-      const formatterSpec = await koqFormatter.getFormatterSpec({
+      const koqFormatter = new KoqPropertyValueFormatter({ schemaContext, formatsProvider: IModelApp.formatsProvider });
```

Comment:
we were intentionally using the deprecated `KoqPropertyValueFormatter` to stay compatible with older versions of iTwin.js core

Reply:
I'm surprised that this didn't cause regression tests to fail, I guess we don't have a full stack test that uses this piece of code.

Reply:
reverted.